library/Process-linux.cpp

```diff
@@ -120,13 +121,18 @@ Process::~Process()
 
 string Process::doReadClassName (void * vptr)
 {
-    //FIXME: BAD!!!!!
-    char * typeinfo = Process::readPtr(((char *)vptr - sizeof(void*)));
-    char * typestring = Process::readPtr(typeinfo + sizeof(void*));
+    char* typeinfo = Process::readPtr(((char *)vptr - sizeof(void*)));
```

Comment:
I think ab9rf is in the process of unifying these files, and uses `diff` to identify the chunks that differ. Changing the formatting at this point would probably cause her pain.

Reply:
eh, don't worry about it. i can tell VSCode to ignore whitespace

Reply:
If desired I can just update styling to be consistent between Linux and Darwin as they are essentially equivalent.

Reply:
> If desired I can just update styling to be consistent between Linux and Darwin as they are essentially equivalent.

Feel free to do as you see best here.